test/base.py

```diff
@@ -1,12 +1,28 @@
 import unittest
-
 import jsonschema
+import os, sys
 
+sys.path.insert(1, os.path.join(sys.path[0], '..'))
+from genson import Schema
 
 class SchemaTestCase(unittest.TestCase):
+
+    def assertGenSchema(self, instance, options, expected):
```

Comment:
I figured out a way to simplify the testing interface using test instance state. I'll add it after the merge.